node/api/client/renter.go

```diff
@@ -93,6 +94,18 @@ func (c *Client) RenterPricesGet() (rpg api.RenterPricesGET, err error) {
 	return
 }
 
+// RenterPostDownloadCacheSize uses the /renter endpoint to change the renter's
+// downloadCacheSize for streaming
+func (c *Client) RenterPostDownloadCacheSize(cacheSize uint64) (err error) {
+	if cacheSize <= 0 {
+		return errors.New("Download Cache Size must be greater than 0")
```

Comment:
The client library doesn't need to do any checks. It should just serialize the inputs provided in safe types to strings and send the request to the API. The cacheSize check only needs to happen on the siad side. `SetSettings` is probably a good place for that.